modules/apps/portal-vulcan/portal-vulcan-test/src/testIntegration/java/com/liferay/portal/vulcan/internal/graphql/servlet/test/BaseGraphQLServlet.java

```diff
@@ -171,6 +171,11 @@ public TestServletData(TestDTO testDTO) {
 			_testQuery = new TestQuery(testDTO);
 		}
 
+		@Override
+		public String getApplicationName() {
```

Comment:
This is because a `TreeMap` cannot have `null` keys.

Reply:
I think it should not be a breaking change, because REST Builder is the one that is implementing the ServletData interface. However, maybe there is a customer that is manually implementing the ServletData interface and then, we would be breaking them. What do you think about removing the default implementation?